assets/js/app/example-app/components/ColorDisplay/ColorDisplay.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const ColorDisplay = (props) => {
+  const { color } = props;
+
+  const style = {
```

Comment:
Have we changed the recommendation of keeping CSS in SASS? If not, maybe we should avoid it in our example component. We could change the `className` instead.

@laisvarejao